doc/source/news/14.md

```diff
@@ -0,0 +1,77 @@
+# News - 14 series
+
+(release-14-04)=
+## Release 14.04 - 2024-06-12
+
+### Improvements
+
+* [{doc}`/install/almalinux`] Added support for MariaDB 10.10.6, 10.11.5, 10.4.33, 10.5.25, 10.6.18, 10.9.8, and 10.11.8.
+
+* Added support for MySQl 8.0.37.
+
+* Added support for MySQl 8.4.
+
+* [{doc}`/install/almalinux`] Added support for Percona Server 5.7.43.
+
+* [{doc}`/install/almalinux`] Added support for Percona Server 8.0.36-28.
+
+* Dropped support MySQL Community Server 5.7.
+
+    Because MySQL Community Server 5.7. reached EOL.
+
+* Dropped support Percona Server 5.7
+
+    Because Percona Server for MySQL 5.7 reached EOL.
+
+* Dropped support for MariaDB 10.7, 10.8, and 10.9.
+
+    Because MariaDB 10.7, 10.8, and 10.9 reached EOL.
+
+* Dropped support for MariaDB 10.10.
+
+    Because the support term of MariaDB 10.10 is short.
+
+* Dropped support for Amazon Linux 2.
+
+    Because Groonga already dropped support for Amazon Linux 2.
+
+* [{doc}`/install/ubuntu`] Dropped support for Ubuntu 20.04 (focal).
+
+* Dropped support for CentOS 7.
+
+    Because CentOS 7 will reach EOL on 2024-05.
+
+* [{doc}`/install/debian`] Dropped support for Debian GNU/Linux bullseye.
+
+    Because Debian GNU/Linux bullseye will reach EOL on 2024-07.
+
+* Dropped support for MariaDB 10.4.
+
+    Because MariaDB 10.4 will reach EOL on 2024-06-18.
+
+* [{doc}`/install/ubuntu`] Added support for Ubuntu 24.04 (noble).
+
+* Mroongaが初期化されていない状態で、MroongaのUDFを使用した場合にエラーを出力するようにしました。[MDEV-21166][Reported by Ian Gilfillan.]
+
+  この修正が入る前は、Mroongaを初期化していない状態（`INSTALL PLUGIN mroonga`を実行していない状態）でMroongaのUDFを使用するとクラッシュしてしまいました。
+  この修正によって、クラッシュではなく、エラーをログに出力するようになります。
+
```

Comment:
```
* Added functionality to output an error when Mroonga's UDF is used while Mroonga is uninitialized. [MDEV-21166][Reported by Ian Gilfillan.]
  Before this fix, using Mroonga's UDF without initializing Mroonga (not having run `INSTALL PLUGIN mroonga`) would cause a crash.
  This patch changes the output from a crash to logging an error.
```